common-lib/resources/ingest-table-of-contents-template.txt

```diff
@@ -0,0 +1 @@
+</ul></li></ul> <li>%uc-plural-generic%<ul><li>/providers/&lt;provider-id&gt;%plural-generic%&lt;native-id&gt;<ul><li><a href="#create-update-%generic%">PUT - Create or update a %generic%.</a></li><li><a href="#delete-%generic%">DELETE - Delete a %uc-generic%.</a></li>
```

Comment:
why does all this not come from the ingest.md headers?

Reply:
More of it can but, I will still need to make some changes. Some of the text such as the "Put" or "Delete" isn't in the header. The "plural-generic" pieces can as well but, they will have to get transformed at specific parts to be plural or uppercase. The providers section must also be in template since it is not in the header